### ISSUE TITLE
Blackbar distance for higher aspect ratio

### ISSUE THREAD

**Issue by subsoap** [rgrams/rendercam#3]
Say I have a game with a 4:3 aspect ratio.

If I resize the window I want it to fit the game into a 4:3 aspect ratio. If the window was resized to a 4:3 aspect ratio then it would clip off the 16:9 aspect ratio area.

But if I resize the window to be larger than 4:3 I want the black bars to not draw up to a for example 16:9 aspect ratio.

![2017-11-07 11_57_21-lnpoverlaywindow](https://user-images.githubusercontent.com/409170/32514755-d1edd0ba-c3b2-11e7-8071-a55b49facb25.png)

The reason for this is to have a game area with a smaller aspect ratio, but still have it easy for the game to look nice on widescreen monitors without the letterboxing so close to the game area.

**Comment by rgrams** [comment]
Hmm . . . sorry I don't follow. With the current fixed aspect ratio mode, if you set it at 4:3 then it will always be 4:3, no matter the window size or proportion. You want something different?

**Comment by subsoap** [comment]
I mean that you can scale the window down to 4:3 and everything will fit without showing any black bars, but you can widen the window up to 16:9 and it would be able to continue to draw game background/particles/whatever and then draw black bars after that. I can make a modification later to show.

**Comment by rgrams** [comment]
Ah, so you want an aspect ratio range?

**Comment by subsoap** [comment]
No, just the ability for the game to fit one aspect but draw black bars to another.
